toolchain/sem_ir/entity_with_params_base.h

```diff
@@ -95,21 +94,21 @@ struct EntityWithParamsBase {
   // A block containing the pattern insts for the parameter lists.
   InstBlockId pattern_block_id;
   // A block containing, for each implicit parameter, a reference to the
-  // instruction in the entity's declaration block that depends on all other
-  // pattern-match insts pertaining to that parameter.
-  InstBlockId implicit_param_refs_id;
-  // A block containing, for each implicit parameter, a reference to the
   // instruction in the entity's pattern block that depends on all other
   // pattern insts pertaining to that parameter.
   InstBlockId implicit_param_patterns_id;
   // A block containing, for each explicit parameter, a reference to the
-  // instruction in the entity's declaration block that depends on all other
-  // pattern-match insts pertaining to that parameter.
-  InstBlockId param_refs_id;
-  // A block containing, for each explicit parameter, a reference to the
   // instruction in the entity's pattern block that depends on all other
   // pattern insts pertaining to that parameter.
   InstBlockId param_patterns_id;
+  // A block containing, for each calling convention parameter (including the
+  // return slot), a reference to the instruction in the function's declaration
+  // block that represents that parameter. These instructions will be in the
+  // `AnyParam` category. This is not populated on imported entities, because it
+  // is relevant only for the function definition.
+  // TODO: can this be moved to `Function`, since it is not applicable to other
+  // kinds of entities?
+  InstBlockId calling_convention_params_id;
```

Comment:
@zygoloid Can you just confirm that this is what you're expecting?

Reply:
Just to note, one thing I'd particularly want to be sure of is that you both understand what "calling convention parameter" is, that it's a term I'm just not familiar with. To me, that sounds distinct from "implicit" or "implicit", not a union. If you're still considering names, I would suggest something like "full parameters" or "signature parameters" for this.

Reply:
See `pattern_match.h` for the definition of that term (such as it is). Do you think it would make sense to move that here?

Reply:
I think this would be the more central location to document a a term. However, I've also brought this up on [#naming](https://discord.com/channels/655572317891461132/963846118964350976/1305655302947082262).